src/App.jsx

```diff
@@ -14,57 +17,91 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const goodsMap = goods.map(good => {
+  return {
+    name: good,
+    id: uuid(good),
```

Comment:
The `uuid` function does not take any arguments. You should call it without passing `good` as an argument. Change `id: uuid(good)` to `id: uuid()`.